src/services/itemMembership/repository.ts

```diff
@@ -677,25 +728,23 @@ export const ItemMembershipRepository = AppDataSource.getRepository(ItemMembersh
    * @param account Account used as `creator` for any new memberships
    * @param newParentItem Parent item to where `item` will be moved to
    */
-  async moveHousekeeping(
-    item: Item,
-    account: Account,
-    newParentItem?: Item,
-  ): Promise<{
-    inserts: Partial<ItemMembership>[];
-    deletes: Partial<ItemMembership>[];
-  }> {
-    if (!newParentItem) return this.detachedMoveHousekeeping(item, account);
+  async moveHousekeeping(item: Item, account: Account, newParentItem?: Item) {
+    if (!newParentItem) {
+      return this.detachedMoveHousekeeping(item, account);
+    }
 
     const { path: newParentItemPath } = newParentItem;
     const index = item.path.lastIndexOf('.');
 
     const parentItemPath = index > -1 ? item.path.slice(0, index) : undefined;
     const itemIdAsPath = index > -1 ? item.path.slice(index + 1) : item.path;
 
-    const rows = await this.query(`
+    const rows = (await this.repository.query(`
```

Comment:
should we also use query builder here? Or it is not possible?

Reply:
It is, but I perfer doing it in another PR